### PR TITLE
spelling: notwithstanding

### LINKED ISSUE #30943: spelling of `Notwithstanding` in content/site-policy/github-terms/github-corporate-terms-of-service.md
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/.github/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/site-policy/github-terms/github-corporate-terms-of-service#r-changes-to-the-service-or-terms

### What part(s) of the article would you like to see updated?

> GitHub changes the Service via Updates and addition of new features. **Nothwithstanding** the foregoing, GitHub reserves the right at any time to modify or discontinue, temporarily or permanently, the Service (or any part of it) with or without notice.

### Additional information

https://github.com/github/docs/pull/30890#discussion_r1441443469

### PR DESCRIPTION
**Pull request by jsoref**
### Why:

Closes: #30943

### Check off the following:

- [x] I have reviewed my changes in staging, available via the **View deployment** link in this PR's timeline.

  - For content changes, you will also see an automatically generated comment with links directly to pages you've modified. The comment won't appear if your PR only edits files in the `data` directory.
- [ ] For content changes, I have completed the [self-review checklist](https://docs.github.com/en/contributing/collaborating-on-github-docs/self-review-checklist).

### CODE REVIEW

**Review by nguyenalex836 — APPROVED** [review]

**Comment by nguyenalex836** [comment]
@jsoref Thanks so much for opening a PR with this fix! I'll update the branch and get this merged once tests are passing 🍏

**Comment by nguyenalex836** [comment]
@jsoref Thanks so much for opening a PR with this fix! I'll update the branch and will need to get an approval from our site policy team - but once that is finished, I'll get this merged 🍏

File: content/site-policy/github-terms/github-corporate-terms-of-service.md
Comment (jsoref) [inline content/site-policy/github-terms/github-corporate-terms-of-service.md:328]: https://docs.github.com/en/site-policy/github-terms/github-corporate-terms-of-service#r-changes-to-the-service-or-terms
image

File: content/site-policy/github-terms/github-corporate-terms-of-service.md
Comment (jsoref) [inline content/site-policy/github-terms/github-corporate-terms-of-service.md:328]: https://docs-30944-c6597d.preview.ghdocs.com/en/site-policy/github-terms/github-corporate-terms-of-service#r-changes-to-the-service-or-terms
image

**Review by margaret-tucker — APPROVED** [review]

**Pull request merged**
